docs/migration/gas-fee-issues.md

```diff
@@ -0,0 +1,19 @@
+---
+sidebar_label: '⛽️ Gas Fee Issues'
```

Comment:
I think it creates duplicated content with

https://support.lukso.network/migration/introduction#im-getting-very-high-gas-fees-when-using-metamask-whats-going-on

So I suggest to remove the content of that link about as it is now in this new page created